docs/Contributing.asciidoc

```diff
@@ -374,12 +392,19 @@ Replace VAR1 and VAR2 in -e switch to match a test battery of the test matrix:
 |DEVELOPER_FULLSTACK=1|
 |============================
 
-==== tips
+So by replacing VAR1 and VAR2 with those values one can trigger the different tests of the matrix.
+
+Of course it is also possible to run (specific) tests directly via `prove` instead of using the Makefile targets.
 
-Running commands will be executed after the initialization script (database creation and so on..). So if there's the need to run an interactive
-session after it just do:
+You might have noticed that there is also the Makefile target `docker-test`. It also invokes
```

Comment:
I would refrain from noting the actual dependencies of make targets within the documentation. Rather rename the targets in the Makefile to be obvious.